src/ddprof_worker.cc

```diff
@@ -264,6 +267,12 @@ static DDRes ddprof_unwind_sample(DDProfContext *ctx, perf_event_sample *sample,
     ddprof_stats_add(STATS_UNWIND_TRUNCATED_INPUT, 1, nullptr);
   }
 
+  if (us->_dwfl_wrapper->_inconsistent) {
```

Comment:
Just curious, what is the reason for moving this block ?

Reply:
So the cleanup before was only removing unwinding information.
Now that we clean up all information related to the pid, we want to do it at a higher level.